### PR TITLE
bug: tfrender now properly associates escalation policy step to oncall schedule

### PR DESCRIPTION
**Pull request by wilsonehusin**
In some scenarios, we would split a schedule from existing provider to multiple schedules in FireHydrant. When that happens, the compromise we can make in escalation policy referring to those schedules is to include both in the target, with expectations that individual teams will then modify their escalation policy steps to the desired behavior.

### CODE REVIEW

File: tfrender/testdata/TestRenderPagerDuty/EscalationPolicy_seed.sql [hunk 1]
```diff
@@ -25,6 +25,6 @@ INSERT INTO ext_escalation_policy_steps VALUES('PKQDFZH','P2D2WR1',0,'PT30M');
 INSERT INTO ext_escalation_policy_steps VALUES('P08T67P','PS6ITO0',0,'PT30M');
 
 INSERT INTO ext_escalation_policy_step_targets VALUES('P08T67P','User','PXI6XNI');
-INSERT INTO ext_escalation_policy_step_targets VALUES('PKQDFZH','OnCallSchedule','PGR96WL-PR3J6XJ');
+INSERT INTO ext_escalation_policy_step_targets VALUES('PKQDFZH','OnCallSchedule','PGR96WL');
```
Comment (wilsonehusin) [inline tfrender/testdata/TestRenderPagerDuty/EscalationPolicy_seed.sql:28]: Simulating a schedule which was only referred to by its prefix.

**Review by AlexisJasso — APPROVED** [review]
This works for me.  Should we consider doing something similar for Opsgenie re: schedules and rotations?

**Comment by wilsonehusin** [comment]
> This works for me. Should we consider doing something similar for Opsgenie re: schedules and rotations?

yeah I _think_ it should work? Opsgenie schedule and rotations are saved as `<schedule_id>-<rotation_id>` right?

**Comment by AlexisJasso** [comment]
> > This works for me. Should we consider doing something similar for Opsgenie re: schedules and rotations?
> 
> yeah I _think_ it should work? Opsgenie schedule and rotations are saved as `<schedule_id>-<rotation_id>` right?

Yep!

**Pull request merged**